release/_publishing.sh

```diff
@@ -1,5 +1,47 @@
 #!/bin/bash
 
+function add_fixed_issues_to_project_version {
+	IFS=',' read -r -a fixed_issues_array < "${_BUILD_DIR}/release/release-notes.txt"
+
+	local fixed_issues_array_length="${#fixed_issues_array[@]}"
+
+	local fixed_issues_array_part_length=$((fixed_issues_array_length / 4))
+
+	local patcherProjectVersionId="${1}"
+
+	for counter in {0..3}
+	do
+		start_index=$((counter * fixed_issues_array_part_length))
+		end_index=$((start_index + fixed_issues_array_part_length))
+
+		if [ "${counter}" -eq 3 ]
+		then
+			end_index="${fixed_issues_array_length}"
+		fi
+
+		IFS=',' fixed_issues="${fixed_issues_array[*]:${start_index}:$((end_index - start_index))}"
```

Comment:
@kiwm, can you please check if we can replace the `$((end_index - start_index))` by `fixed_issues_array_part_length` only?